torchgeo/datamodules/chesapeake.py

```diff
@@ -169,24 +169,34 @@ def preprocess(self, sample: Dict[str, Any]) -> Dict[str, Any]:
         Returns:
             preprocessed sample
         """
-        sample["image"] = sample["image"] / 255.0
-        sample["mask"] = sample["mask"].squeeze()
+        sample["image"] = sample["image"].float()
+        sample["image"] /= 255.0
+
+        if "mask" in sample:
```

Comment:
Added this type of check in order to be able to re-use the `preprocess(...)` logic in cases where "masks"/"labels" aren't available.